src/app/tutorials/tutorials.router.ts

```diff
@@ -1,9 +1,16 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { TutorialsComponent } from './tutorials.component';
+import { InputComponent } from './input/input.component';
 
 const routes: Routes = [
-  { path: '', component: TutorialsComponent },
+  {
+    path: '',
+    component: TutorialsComponent,
+    children: [
+      { path: 'input', component: InputComponent },
```

Comment:
This sub-module is loaded by the path `tutorials` , so path here is going to be `tutorials/input` and will render the component `InputComponent`